config/example.toml

```
@@ -27,6 +27,10 @@ ci_passed_label = "ci-passed"
 # have approved.
 reviewed_label = "reviewed"
 
+# Optional: Label that can be manually added to PRs to not block on reviews, for trivial changes.
+# If there's also a `block_merge_label` set, it has priority over this label being set.
+trivial_review_label = "trivial"
```

Can we call it this instead, think it is more explicit what the operation does and it sounds a bit scarier than a "trivial" change so devs hopefully take a bit more care into putting the label on it.    in our internal repos we will likely call this `s: skip review`

can we also have it not be active by default by not having it set? as we likely want to have it disabled by default in all repos using octobors and have them opt into it. one likely also want to only use it for private repos with trusted devs, rather than any open source repos

```suggestion
skip_review_label = ""
```

---

> Can we call it this instead, think it is more explicit what the operation does and it sounds a bit scarier than a "trivial" change so devs hopefully take a bit more care into putting the label on it. in our internal repos we will likely call this s: skip review

Happy to rename this, but according to the _specification and implemented behavior_: this will _not_ ignore a review that requested changes, so to some extent it does not override existing reviews. So "skip" didn't quite feel right to me. I don't have a very strong opinion, so if people preferred it we can use that instead (or anything else we'd come up with and agree upon).

> can we also have it not be active by default by not having it set?

I really hope that people are tweaking the values in the example.toml file before using it :sweat_smile: as it's rather intended to show all the possible options and their values. I'll comment this line in the file.

---

>  this will not ignore a review that requested changes, so to some extent it does not override existing reviews. So "skip" didn't quite feel right to me.

I guess technically it is "skip additional reviews" or "do not wait for reviews". think "skip" can work okay enough though and is more explicit than the previous "trivial" at least. with all labels we can have a longer description for what it means and does also. for the current `s: skip review` I have this as description (which we can update):

"This is a trivial change that will be automerged as soon as CI passes"